docker/ckan/2.9.9.Dockerfile

```diff
@@ -19,7 +19,11 @@ RUN echo "pip install ckanext-datagovuk..." && \
 
     # install ckanext-datagovuk
     pip install $pipopt -U -r requirements.txt && \
-    pip install $pipopt -U -e . 
+    pip install $pipopt -U -e . && \
+
+    # pin lxml as 5.1.1 removes required property _ElementStringResult
+    # TODO: revisit for CKAN 2.10 upgrade as might not be required
+    pip install lxml<5.1.1
```

Comment:
Did you mean to do this in requirements.txt? Seems confusing to be overriding it here.

Reply:
Could do it in the ckanext-spatial requirements but also need to investigate whether the upgrade to 2.10 fixes this issue which might delay this fix. So I'm ok with just pinning it here for now

Reply:
No worries 👍